documentation/src/main/asciidoc/reference/backend-elasticsearch.asciidoc

```diff
@@ -907,6 +907,9 @@ or disabling `doc_values` on a field used for sorting.
 An invalid custom mapping may not trigger any exception on bootstrap, but later while indexing or querying.
 In the worst case, it could not trigger any exception, but simply lead to incorrect search results.
 Exercise extreme caution.
+
+<<schema-management-export-filesystem, Exporting the schema based on the mapping information>> might be a good first step
+for further customization of it.
```

Comment:
Mmm ok 
Well, my idea here was that if some parts of mapping are hard to get:
>I think dynamic_templates should be retained if it’s not mentioned in the custom mapping, because it’s complex to get right manually

then if the user exports what HS has generated without providing their own mapping file - they could then get the bits they need and drop the remaining things, or modify or do whatever they wish 😃. 
But I'm fine dropping this line if it might confuse others 😃